wizard/migrations/0079_challengemodel_build_at.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10 on 2017-06-22 22:12
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.utils.timezone
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('wizard', '0078_datasetmodel_updated_at'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='challengemodel',
+            name='build_at',
```

Comment:
Be careful to not make too many migrations, can get unwieldy after too long :) Try to think out the schema ahead of time, although that's often very difficult/impossible, so we have as few migrations as possible.